docs/guides/ci-cd.md

```diff
@@ -0,0 +1,251 @@
+---
+description: CI/CD resources for working with Prefect.
+tags:
+    - CI/CD
+    - continuous integration
+    - continuous delivery
+search:
+  boost: 2
+---
+
+# CI/CD with Prefect
+
+Prefect is used by many organizations in their CI/CD pipelines.
```

Comment:
I have a weakly held preference for using the word "process" instead of "pipeline" in order to avoid confusion with most user's association with "data pipeline".